airflow-core/src/airflow/ui/src/layouts/Details/DetailsLayout.tsx

```diff
@@ -79,7 +79,7 @@ export const DetailsLayout = ({ children, error, isLoading, tabs }: Props) => {
       </HStack>
       <Toaster />
       <BackfillBanner dagId={dagId} />
-      <Box flex={1} minH={0}>
+      <Box data-testid="debug-box" flex={1} minH={0}>
```

Comment:
What are these for?

Reply:
ha!

that was me trying to figure out what went where as i was stumbling through this. i will remove